lumigator/python/mzai/evaluator/tests/integration/test_lm_harness.py

```diff
@@ -24,7 +24,7 @@ def job_config(llm_model_path) -> LMHarnessJobConfig:
 
 
 def test_lm_harness_job(job_config):
-    buddy = LMBuddy()
+    buddy = Evaluator()
     result = buddy.evaluate(job_config)
```

Comment:
Note that this will also be addressed here https://github.com/mozilla-ai/lumigator/compare/LUMI-69...LUMI-69-prometheus?expand=1#diff-10e0316f3138c092d3e1790c2bdff4259d65b18d70f854239f41482ceb37ae5bR28